cmd/logger/audit.go

```diff
@@ -27,23 +30,43 @@ import (
 // ResponseWriter - is a wrapper to trap the http response status code.
 type ResponseWriter struct {
 	http.ResponseWriter
-	statusCode      int
+	StatusCode      int
 	startTime       time.Time
 	timeToFirstByte time.Duration
```

Comment:
btw we should use this in Trace.Info() output timeToFirstByte, atleast as part of the struct and provided in --json output